tests/test_execution.py

```diff
@@ -60,21 +64,46 @@ def test_simple_execution_context():
     assert not context.alive
     assert not context.started
     assert not context.stopped
+    assert not context.killed
+    assert not context.defunct
 
     context.write(BEGIN, (), END)
 
     assert not context.alive
     assert not context.started
     assert not context.stopped
+    assert not context.killed
+    assert not context.defunct
 
     context.start()
 
     assert context.alive
     assert context.started
     assert not context.stopped
+    assert not context.killed
+    assert not context.defunct
 
     context.stop()
 
     assert not context.alive
     assert context.started
     assert context.stopped
+    assert not context.killed
+    assert not context.defunct
+
+
+def test_simple_error_context():
+    graph = Graph()
+    graph.add_chain(*bad_chain)
+
+    context = GraphExecutionContext(graph)
+    context.write(BEGIN, (), END)
+    context.start()
+
+    assert context.alive
+    assert context.started
+
+    # How to run graph?
```

Comment:
Confused on how to get the context to actually run the nodes. Sorry for the noob questions